README.rst

```diff
@@ -939,6 +1018,216 @@ configuration status of a device changes, this ensures the check reacts
 quickly to events happening in the network and informs the user promptly
 if there's anything that is not working as intended.
 
+Iperf
+~~~~~
+
+This check provides network performance measurements such as maximum achievable bandwidth, 
+jitter, datagram loss etc of the device using `iperf3 utility <https://iperf.fr/>`_. 
+
+This check is **disabled by default**. You can enable auto creation of this check by setting the
+`OPENWISP_MONITORING_AUTO_IPERF <#OPENWISP_MONITORING_AUTO_IPERF>`_ to ``True``.
+
+It also supports tuning of various parameters.
+
+You can also change the parameters used for iperf checks (e.g. timing, port, username, password, rsa_publc_key etc) 
```

Comment:
please break lines and maintain line length consistent with the rest of the document